src/BrightScriptDebugSession.ts

```diff
@@ -287,6 +306,157 @@ export class BrightScriptDebugSession extends DebugSession {
         }
     }
 
+    private componentLibraryPostfix: string = '__lib';
+
+    protected async prepareAndHostComponentLibraries(componentLibraries, componentLibrariesOutDir: string, port: number) {
+        if (componentLibraries && componentLibrariesOutDir) {
+            this.componentLibrariesOutDir = componentLibrariesOutDir;
+            this.componentLibrariesStagingDirPaths = [];
+            let replace = require('replace-in-file');
```

Comment:
Move this to the top of the file with the other imports, and use the typescript import syntax. 
```typescript
import * as replaceInFile from 'replace-in-file';
```

Reply:
Moving to this type of import seems to be giving me a problem. 
![image](https://user-images.githubusercontent.com/9591618/59078645-6e738280-88b6-11e9-9894-2ee93534629c.png)

Reply:
Not sure really why that is though.